axelrod/strategies/cycler.py

```diff
@@ -72,9 +72,11 @@ def __init__(self, cycle="CCD") -> None:
         self.classifier['memory_depth'] = len(cycle) - 1
 
     def strategy(self, opponent: Player) -> Action:
+        actions = {'C': Actions.C, 'D': Actions.D}
         curent_round = len(self.history)
         index = curent_round % len(self.cycle)
-        return self.cycle[index]
+        action_str = self.cycle[index]
+        return actions[action_str]
```

Comment:
there is a funky thing going on with inheritance.  Cycler has several child classes.  I'm pretty sure this error triggered on the children but not the parent.  there are 6 children and 12 errors.  so i wrote a new cycler that inherits from MockPlayer and got the same issue.  here's the code for child of MockPlayer.

```
class Cycler(MockPlayer):
    name = 'Cycler'
    classifier = {
        'memory_depth': 1,
        'stochastic': False,
        'makes_use_of': set(),
        'long_run_time': False,
        'inspects_source': False,
        'manipulates_source': False,
        'manipulates_state': False
    }
    
    def __init__(self, cycle):
        for_mock = []
        for char in cycle:
            if char == 'C':
                for_mock.append(Actions.C)
            else:
                for_mock.append(Actions.D)
        super(Cycler, self).__init__(for_mock)
```

do you have any idea what's going on?  i'll bet anything there's some issue with the way the children inherit but i don't know what it is.